### PR TITLE
More engines!

### PR DESCRIPTION
**Pull request by cgalvarez**
Adds more engines to JavaScript, PHP, Markdown, Python.

Also adds engines for CSS and shell (which I miss a lot!).

### CODE REVIEW

**Review by Arcanemagus — APPROVED** [review]
Syntax looks good to me, I'll leave whether those are valid to @maxjacobson to comment on!

File: lib/index.js [hunk 1]
```diff
@@ -121,13 +121,18 @@ export default {
       Ruby: ['rubocop', 'reek'],
       'Ruby on Rails': ['rubocop', 'reek'],
       'Ruby on Rails (RJS)': ['rubocop', 'reek'],
-      JavaScript: ['eslint'],
+      JavaScript: ['checks', 'eslint', 'nodesecurity', 'requiresafe'],
```
Comment (maxjacobson) [inline lib/index.js]: for now, will you remove `checks`? It's not quite ready for public consumption

**Review by maxjacobson — CHANGES_REQUESTED** [review]

**Comment by maxjacobson** [comment]
@Arcanemagus Thanks for the ping. Btw, will you add @ale7714 to the codeclimate-specific team? She can help out with giving CC-specific perspective and review

**Comment by Arcanemagus** [comment]
Promoted you to team maintainer, does that give you permissions to add people from outside the org, or just ones already in the org?

**Comment by maxjacobson** [comment]
@Arcanemagus Ah, thanks. Only ones already in the org though

**Comment by Arcanemagus** [comment]
Ah, dang, I'll add her then 😉.

**Comment by cgalvarez** [comment]
@maxjacobson `checks` removed :wink:

File: lib/index.js [hunk 2]
```diff
@@ -121,13 +121,18 @@ export default {
       Ruby: ['rubocop', 'reek'],
       'Ruby on Rails': ['rubocop', 'reek'],
       'Ruby on Rails (RJS)': ['rubocop', 'reek'],
```
Comment (maxjacobson) [inline lib/index.js]: will you also please add `flog` to these three ruby-related mappings?

**Review by maxjacobson — CHANGES_REQUESTED** [review]

File: lib/index.js (hunk 2 above)
Comment (cgalvarez) [reply lib/index.js]: Sure. Already sent. There you have'em.

**Review by maxjacobson — APPROVED** [review]

**Pull request merged**